src/lighteval/tasks/lighteval_task.py

```diff
@@ -602,6 +623,47 @@ def download_dataset_worker(args):
     return dataset
 
 
+def create_multi_turn_contexts(
```

Comment:
This should be managed by the fewshot manager, since it manages all contexts and prompt creations atm (could be renamed prompt manager btw)